### PR TITLE
feat(oianalytics): manage South items import from OIAnalytics

### PR DESCRIPTION
**Pull request by burgerni10**

### CODE REVIEW

File: backend/src/model/oianalytics-command.model.ts [hunk 1]
```diff
@@ -88,6 +88,16 @@ export interface OIBusDeleteNorthConnectorCommand extends BaseOIBusCommand {
...
 }
 
+export interface OIBusCreateOrUpdateSouthConnectorItemsFromCSVCommand extends BaseOIBusCommand {
+  type: 'create-or-update-south-items-from-csv';
+  southConnectorId: string;
+  commandContent: {
+    deleteItemsNotPresent: boolean;
+    csvContent: string;
+    delimiter: string;
+  };
```
Comment (marouanehassanioptimistik) [inline backend/src/model/oianalytics-command.model.ts:98]: it is not wrapped in a commandContent is it?

**Review by marouanehassanioptimistik — CHANGES_REQUESTED** [review]

File: backend/src/model/oianalytics-command.model.ts (hunk 1 above)
Comment (burgerni10) [reply backend/src/model/oianalytics-command.model.ts:98]: Not from OIAnalytics, but I wrap it in OIBus when received, in a similar fashion than OIBus update commands. Should I created dedicated fields instead (for both commands)?

Comment (marouanehassanioptimistik) [reply backend/src/model/oianalytics-command.model.ts:98]: not sure why you would transform the original signature, what does this bring?

Comment (marouanehassanioptimistik) [reply backend/src/model/oianalytics-command.model.ts:98]: nevermind it is oibus model forget what i said

**Review by marouanehassanioptimistik — APPROVED** [review]

**Pull request merged**